clients/java/src/main/java/io/odpf/raccoon/model/Result.java

```diff
@@ -0,0 +1,33 @@
+package io.odpf.raccoon.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder(toBuilder = true)
+
+/*
+ * Result is the wrapper class for the raccoon client response,
+ * It holds the status of the request, error message and the response.
+ */
+public class Result<T> {
```

Comment:
Isn't this wrapper redundant of Response? Do we need this?

Reply:
yeah for now, I was only thinking about the future, if we introduce the new API and make the schema changes.

np, I will update.